docs/content/users/extend/customizing-images.md

```diff
@@ -101,14 +101,16 @@ These files’ content will be inserted into the constructed Dockerfile for each
 
 For certain use cases, you might need to add directives very early on the Dockerfile like proxy settings or SSL termination. You can use `pre.` variants for this that are inserted *before* everything else:
 
+* `.ddev/web-build/pre.Dockerfile`
 * `.ddev/web-build/pre.Dockerfile.*`
+* `.ddev/db-build/pre.Dockerfile`
 * `.ddev/db-build/pre.Dockerfile.*`
```

Comment:
nit: Let's put the `.*` first, best practice. People are better off to name these things. 

```suggestion
* `.ddev/web-build/pre.Dockerfile.*`
* `.ddev/web-build/pre.Dockerfile`
* `.ddev/db-build/pre.Dockerfile.*`
* `.ddev/db-build/pre.Dockerfile`
```